sdk/storageactions/Azure.ResourceManager.StorageActions/src/Generated/ProviderConstants.cs

```diff
@@ -9,8 +9,9 @@
 
 namespace Azure.ResourceManager.StorageActions
 {
-    internal static class ProviderConstants
+    internal static partial class ProviderConstants
     {
-        public static string DefaultProviderNamespace { get; } = ClientDiagnostics.GetResourceProviderNamespace(typeof(ProviderConstants).Assembly);
+        /// <summary> Gets the DefaultProviderNamespace. </summary>
+        public static string DefaultProviderNamespace => ClientDiagnostics.GetResourceProviderNamespace(typeof(ProviderConstants).Assembly);
```

Comment:
I kind of think we should use the old way - auto property with an initialization value.